chat-components/src/androidTest/java/com/pubnub/components/ChannelListTest.kt

```diff
@@ -65,20 +66,19 @@ class ChannelListTest : BaseTest() {
             assertIsDisplayed()
 
             channels.forEachIndexed { index, item ->
-                onChildAt(index).apply {
-                    assertHasClickAction()
-                    onChildren().apply {
-                        assertAny(hasText(item.name))
-                        assertAny(hasText(item.description!!))
-                    }
-                }
+                // Thumbnail
+                onChildAt(3 * index).assertContentDescriptionEquals(InstrumentationRegistry.getInstrumentation().context.getString(R.string.thumbnail))
```

Comment:
Can we extract this (`InstrumentationRegistry.getInstrumentation().context.getString(...)`) to some helper function? Looks like repeating pattern and it is very noisy?

Reply:
Nice catch, will extract it into getter (in `BaseTest` class)